src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1831,7 +1831,7 @@ AletheProofPostprocess::~AletheProofPostprocess() {}
 void AletheProofPostprocess::process(std::shared_ptr<ProofNode> pf)
 {
   // Translate proof node
-  ProofNodeUpdater updater(d_pnm, d_cb, true, false);
+  ProofNodeUpdater updater(d_pnm, d_cb, false, false);
```

Comment:
The post-visit update should already be called as soon as it is defined in the alethe post processor (as well as the shouldUpdatePost, which I added in #8431. 

This change here is only making the updater *not* merge subproofs. Was that leading to issues?

Reply:
This was the change you suggested on Zulip, wasn't it?

Reply:
It hasn't.

Reply:
I will find out why exactly